build.sbt

```diff
@@ -1,25 +1,42 @@
 lazy val fuuid = project.in(file("."))
   .settings(commonSettings, releaseSettings, skipOnPublishSettings)
-  .aggregate(core, docs)
+  .aggregate(core, http4s, docs)
 
 lazy val core = project.in(file("modules/core"))
     .settings(commonSettings, releaseSettings)
     .settings(
       name := "fuuid"
     )
 
+lazy val http4s = project.in(file("modules/http4s"))
+  .settings(commonSettings, releaseSettings)
+  .settings(
+    name := "fuuid-http4s",
+    libraryDependencies ++= Seq(
+      "org.http4s" %% "http4s-dsl" % http4sV % Test
+    )
+  )
+  .dependsOn(`core` % "compile->compile;test->test")
+
 lazy val docs = project.in(file("modules/docs"))
   .settings(commonSettings, skipOnPublishSettings, micrositeSettings)
+  .settings(
+    libraryDependencies ++= Seq(
+      "org.http4s" %% "http4s-dsl" % http4sV
+    )
+  )
   .enablePlugins(MicrositesPlugin)
   .enablePlugins(TutPlugin)
-  .dependsOn(core)
+  .dependsOn(core, http4s)
 
 val catsV = "1.1.0"
 val catsEffectV = "0.10.1"
 val specs2V = "4.3.2"
+val http4sV = "0.18.5"
```

Comment:
Please update the http4s version to `0.18.15`

Reply:
My bad, the version was a typo, I write `5` instead of `15`.